internal/operators/lvm/lvm_operator.go

```diff
@@ -21,6 +22,8 @@ type operator struct {
 	extracter oc.Extracter
 }
 
+const defaultStorageClassName = "lvms-vg1"
```

Comment:
@omertuc do you know if there is a better way to have the default storage class name? It depends on the LVMcluster manifest specs, as it stands it generates as above, but IMO it's very fragile.

Reply:
same is true, if not worse, for odf

Reply:
Is this not something we can set when deploying the operators?

I don't know much about them but it seems like that should be a feature :shrug:

Reply:
Unfortunately I am not aware of any mechanism that would allow me to do that....
The manifest for `AgentServiceConfig` is being sent to the controller, together with `LVMCluster` manifest and similar resources for ODF operator. These latter two are the one responsible to eventually creating the storage classes, and we can't really know the storage class beforehand, it's worked out from the parameters we set.

It's a shitty solution (hardcoded values for storage class) but I haven't found a better way - WDYT?